tools/linting/bad-urls.js

```diff
@@ -21,6 +21,9 @@ const siteData = require('../../src/site/_data/site');
 
 module.exports = rule('remark-lint:bad-urls', checkURL);
 
+// Based on this Stack Overflow answer: https://bit.ly/3ELKYq3
+const locale = /^\/[a-z]{2,3}-[a-z0-9]{2,4}(-[a-z]{2})?/i;
+
```

Comment:
I think ^ enforces the beginning of the string, so this regex will catch only relative urls starting with /locale/

Reply:
`^` does enforce the beginning of the string, but that works here since `parsed.pathname` doesn't include the URL, just the path.